### ISSUE TITLE
Reduced sharing of interests

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7320]
**Preferred term label:**
Reduced sharing of interests

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Abnormal social behavior
HP:0012433

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
added as HP:4000084
